src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -382,6 +382,92 @@ class DemoLocalizationsDelegate extends LocalizationsDelegate<DemoLocalizations>
 }
 {% endprettify %}
 
+<a name="adding-language"></a>
+## Adding support for a new language
+
+An app that needs to support a language that's not included in
+[GlobalMaterialLocalizations]({{site.api}}/flutter/flutter_localizations/GlobalMaterialLocalizations-class.html)
+will have to do some extra work. It's not difficult, however to
+do the complete job one must provide about 70 translations
```

Comment:
We don't use "one" to refer to a person. Instead, we generally use "you" (unless something else is more appropriate). Also, we avoid unnecessary future tense ("will").

How about (modify these words as needed to be clear and fit in with the rest of the doc):

An app...has to do some extra work: it must provide about 70 ...